src/NuGet.Clients/NuGet.VisualStudio.Contracts/NuGet.VisualStudio.Contracts.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <TargetFramework>$(NetStandardVersion)</TargetFramework>
+    <SkipShared>true</SkipShared>
+    <PackProject>true</PackProject>
+    <Shipping>true</Shipping>
+    <IncludeInVsix>true</IncludeInVsix>
+    <IncludeNuGetSharedFiles>true</IncludeNuGetSharedFiles>
+    <RootNamespace>NuGet.VisualStudio.Contracts</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ServiceHub.Framework" Version="$(ServiceHubFrameworkVersion)" NoWarn="NU1605"/>
```

Comment:
Remove the version declaration.

Reply:
oops. I wonder why this didn't cause failures locally.